### ISSUE TITLE
Need an image insert module on API site

### ISSUE THREAD

**Issue by docwilmot** [backdrop-ops/api.backdropcms.org#24]
as title

**Comment by jenlampton** [comment]
Now that inserting images is possible in core, renaming. Also, is this done automatically now, or do we need to configure anything?

**Comment by jenlampton** [comment]
Aha... it looks like BUeditor does not have support for file uploads, since we wrote that directly into CKEditor in core. Updating issue title.

**Comment by BWPanda** [comment]
Or just replace BUEditor for consistency: https://github.com/backdrop-ops/backdropcms.org/issues/613

**Comment by jenlampton** [comment]
On the API site we write a lot more code than we do on the other sites so keeping BUeditor is important. Perhaps what we need is an additional text format with the rich-text editor (and image-uploader). When necessary we could switch between them to upload an image?

**Comment by BWPanda** [comment]
> On the API site we write a lot more code than we do on the other sites so keeping BUeditor is important.

Can you explain this please? What makes BU better for writing code than CK?

If BU stays, maybe it should be added as an option to other sites too, for consistency...

**Comment by jenlampton** [comment]
> What makes BU better for writing code than CK?

When I am typing the HTML code it is nice to have a button to create my starting/ending UL/LI tags for me. This is what I use BU for mostly.

CK makes decisions about where I want those tags based on it being a visual editor, and it assumes that I don't know where to put my own HTML tags. This means I usually spend a lot of time fighting with it: switching back and forth between the visual mode and the HTML code mode to get my tags in the right places. When you are in the HTML code mode, the formatting is really hard to work with, and when you switch back and forth, sometimes things change that you don't want changed.

In short: for pages where you don't need (or want) the visual-editor, cke a real pain to work with.  I find it most helpful on pages that are mostly paragraphs, pages like you find on the b.org site and the forum.

> If BU stays, maybe it should be added as an option to other sites too, for consistency...

Now there's an idea! :)
